### PR TITLE
Fourth-module/first-task

### PR DESCRIPTION
**Pull request by PrimoTibalt**

### CODE REVIEW

File: ModuleFourTasks/Task1/Program.cs [hunk 1]
```diff
@@ -7,7 +7,19 @@ internal class Program
...
         private static void Main(string[] args)
         {
+            const string finishWords = @"Click any button to close application.";
+            if (args.Length < 1)
+            {
+                Console.WriteLine("No arguments were provided. Try again...");
+            }
+            else
+            {
+                args.ToList().ForEach((item) => Console.Write(item[0]));
```
Comment (alexander-trishin) [inline ModuleFourTasks/Task1/Program.cs]: The program should handle user input and show the first character of **each row entered**. Use `Console.Read` or `Console.ReadKey` for that.

**Review by alexander-trishin — CHANGES_REQUESTED** [review]
Hi Anton,

The task was a bit different, please fix your solution.

File: ModuleFourTasks/Task1/Program.cs (hunk 1 above)
Comment (PrimoTibalt) [reply ModuleFourTasks/Task1/Program.cs]: Fixed in e6b279e.

**Review by alexander-trishin — CHANGES_REQUESTED** [review]
Hi Anton,

It's a little better, but again not fit the task given 😄 

The task was the following: Write a console application that displays **the first character** of **each input line**.

This means that we need to:
- read all entered lines until user interrupts input with a key combination (for example, ctrl+c)
- print the first character of each line
- handle situations where user entered an empty string

Consider to check `Console.ReadKey(true)` method.

**Comment by PrimoTibalt** [comment]
> Hi Anton,
> 
> It's a little better, but again not fit the task given 😄
> 
> The task was the following: Write a console application that displays **the first character** of **each input line**.
> 
> This means that we need to:
> 
> * read all entered lines until user interrupts input with a key combination (for example, ctrl+c)
> * print the first character of each line
> * handle situations where user entered an empty string
> 
> Consider to check `Console.ReadKey(true)` method.

Fixed in 4e673ce.

**Review by alexander-trishin — APPROVED** [review]
To be honest, the task is still not completed, because there was nothing about "words"in it 😃 

It was required to print the first letter of each LINE.

Please, be more careful when starting to implement any task.

**Pull request merged**